client-app/src/pages/init/views/Error.vue

```diff
@@ -0,0 +1,66 @@
+<template>
```

Comment:
Error type and components are the part of common functionality, it will better place it on the top level in /common/components folder.

Reply:
Fixed